Classes/Notifications/NotificationsViewController.swift

```diff
@@ -47,53 +64,76 @@ FlatCacheListener {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        resetLeftBarItem()
-        if !showRead {
-            navigationItem.rightBarButtonItem = UIBarButtonItem(
-                title: NSLocalizedString("Archives", comment: ""),
+        resetRightBarItem()
+
+        switch inboxType {
+        case .unread:
+            let item = UIBarButtonItem(
+                image: UIImage(named: "bullets-hollow"),
                 style: .plain,
                 target: self,
-                action: #selector(NotificationsViewController.onViewAll)
+                action: #selector(NotificationsViewController.onMore)
             )
+            item.accessibilityLabel = NSLocalizedString("More options", comment: "")
+            navigationItem.leftBarButtonItem = item
+        case .repo, .all: break
         }
 
         navigationController?.tabBarItem.badgeColor = Styles.Colors.Red.medium.color
     }
 
     // MARK: Private API
 
-    @objc func onViewAll() {
-        let controller = NotificationsViewController(client: client, showRead: true)
-        controller.title = NSLocalizedString("Archives", comment: "")
+    @objc func onMore() {
+        let alert = UIAlertController.configured(preferredStyle: .actionSheet)
+
+        alert.add(action: UIAlertAction(
+            title: NSLocalizedString("View Read", comment: ""),
```

Comment:
Also commented in the PR itself, but why did you rename this?

Reply:
Tbh I'm not sure, lol. I'm going to change it back and be "View Archived"